src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerUpdateTranscriptionApprovedIntTest.java

```diff
@@ -86,12 +90,20 @@ void beforeEach() {
         when(mockUserIdentity.getEmailAddress()).thenReturn(testUser.getEmailAddress());
         when(mockUserIdentity.getUserAccount()).thenReturn(testUser);
         testUserId = testUser.getId();
+
+
     }
 
     @Test
     @Transactional
     void updateTranscriptionApprovedWithoutComment() throws Exception {
 
+        Optional<TranscriptionEntity> existingTranscription = dartsDatabaseStub.getTranscriptionRepository().findById(
```

Comment:
I guess this block could be simplified to avoid Optional here like on line 126 - which is what I prefer.
If you decide to update - I guess the empty lines above on line 93, 94 could be tidied up also

Reply:
done